src/clone/clone-utils.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace EE\Site\Cloner\Utils;
+
+use EE;
+use EE\Site\Cloner\Site;
+use function EE\Utils\get_flag_value;
+use function EE\Utils\random_password;
+use function EE\Utils\trailingslashit;
+
+function copy_site_db( Site $source, Site $destination ) {
+
+	$site_type = $source->site_details['site_type'];
+	$db_host   = $source->site_details['db_host'];
+
+	if ( 'wp' === $site_type || 'php' === $site_type && ! empty( $db_host ) ) {
```

Comment:
Code written inside this `if` condition is using wp-cli, which will fail when site type is `php`.